test/inspector/inspector.status

```diff
@@ -1,16 +0,0 @@
-prefix inspector
-
-# To mark a test as flaky, list the test name in the appropriate section
-# below, without ".js", followed by ": PASS,FLAKY". Example:
-# sample-test                        : PASS,FLAKY
-
-[true] # This section applies to all platforms
-
-[$system==aix]
-test-stop-profile-after-done: PASS, FLAKY
```

Comment:
Shouldn't these be moved to `test/parallel/parallel.status`

Reply:
Good catch -- thank you!